packages/react-client/src/ReactFlightClient.js

```diff
@@ -628,21 +628,60 @@ export function resolveSymbol(
   chunks.set(id, createInitializedChunk(response, Symbol.for(name)));
 }
 
-export function resolveError(
+type ErrorWithDigest = Error & {_digest?: string};
+export function resolveErrorProd(
   response: Response,
   id: number,
+  digest: string,
+): void {
+  if (__DEV__) {
+    // These errors should never make it into a build so we don't need to encode them in codes.json
+    // eslint-disable-next-line react-internal/prod-error-codes
+    throw new Error(
+      'resolveErrorProd should never be called in development mode. Use resolveErrorDev instead. This is a bug in React.',
+    );
+  }
+  const error = new Error('An error occurred in the Server Components render.');
```

Comment:
Not sure the verbiage we want here for prod errors but since this is a generic error resolver we either need to pass in callsite specific prod-ready error messages or use something generic like this

Reply:
It's probably worth saying something about that it's intentionally excluded in prod. Ideally linking to a docs page explaining how to fix it.